ReactWindows/ReactNative.Net46/Views/Text/ReactTextViewManager.cs

```diff
@@ -46,7 +46,7 @@ public void SetColor(RichTextBox view, uint? color)
         /// <param name="view">The view.</param>
         /// <param name="selectable">A flag indicating whether or not the text is selectable.</param>
         [ReactProp("selectable")]
-        public void SetSelectable(RichTextBox view, bool selectable)
+        public void SetSelectable(TextBlock view, bool selectable)
         {
             // ToDo: Manually control selectable
         }
```

Comment:
I checked, and no, they don't appear to have an option to enable/disable selectable.